core-services/prow/02_config/_plugins.yaml

```diff
@@ -1802,6 +1802,8 @@ plugins:
   - approve
```

Comment:
You didn't add this to any automatic merges. Did you use `make new-repo`?

Reply:
No, indeed.... this is new sorcery I was not aware of! Cool